src/command/help.rs

```diff
@@ -334,17 +334,14 @@ mod test {
     fn invalid_command_invocation() -> Result<()> {
         run_help_test(
             vec![String::from("DNE")],
-            String::from(
-                "\
-                command failed to parse: \'DNE\' is not a recognized command.\n\n\t \
-
-
-                    => expected one of \'leaf\' or \'foo\'.\n\n\
-
-
-                Run \'helptree\' for more info on the entire command tree.\n\
-                ",
-            ),
+            r#"command failed to parse: 'DNE' is not a recognized command.
```

Comment:
Nice change here! I have since written more Rust (this was my first crate if I recall), but not enough that I'd call myself a veteran. Is this a common pattern for handling multi-line strings? If you don't mind, I would love to learn more if so. I know there's `indoc!` ([link](https://crates.io/crates/indoc)) though I don't think we need to bring in a dependency just to solve this problem.

Reply:
Thanks for suggesting `indoc!`. 

I am not sure if this is a standard approach. This is just a method I have used in some of my projects to have control over the indentation levels with a special character like `@` to align the start of the line, and I thought it was simple and intuitive and keeps the entire string together without much alterations or escaping of its content because of `r##`. It just has the side effect of including even the spaces before @, so I filter those out.

Let me know if this is okay or you would prefer a different approach.